src/store/modules/picklist/actions.ts

```diff
@@ -19,6 +19,8 @@ const actions: ActionTree<PicklistState, RootState> = {
       if (resp.status === 200 && resp.data.pickingList && !hasError(resp)) {
         commit(types.PICKLISTS_UPDATED, { list: resp.data.pickingList })
         return resp.data;
+      } else if(!resp.data.pickingList) {
+        showToast(translate('No picklist found'));
```

Comment:
We will get `No picklist found` message even when api fails.

Reply:
Added the check sir